app/cdap/components/ConnectionList/Components/ConnectionTabs/index.tsx

```diff
@@ -103,32 +107,32 @@ export default function ConnectionsTabs({
                   index > 1 ? (
                     connectorType.canBrowse ? (
                       <TabLabelCanBrowse
-                        label={connectorType.name}
-                        count={index === 0 ? connectorType.count : undefined}
+                        label={connectorType.name as string}
+                        count={index === 0 ? (connectorType.count as number) : undefined}
```

Comment:
i think, on line 111 we can remove the index === 0 check here, as we have check for index>1 in line 107